ixofoundation/ixo-world
# feat: add deliver and invest pages (#74)

## Scope
- Deliver 
https://nonaredteam.atlassian.net/secure/RapidBoard.jspa?rapidView=77&projectKey=IXO&view=planning&selectedIssue=IXO-418&issueLimit=100

- Invest
https://nonaredteam.atlassian.net/secure/RapidBoard.jspa?rapidView=77&projectKey=IXO&view=planning&selectedIssue=IXO-429&issueLimit=100

## Work done
Added deliver and invest page and created some reusable components, Banner, Benefits and Member

## Steps to test
Compare Deliver and Invest pages to designs (Pages @ http://localhost:XXXX/getixowallet/deliver and .../invest)
https://www.figma.com/file/ok50elV5abdYseQYKaSioG/ixo.world-website-revisions?node-id=5%3A20

Verify links work and go to destinations specified in Jira task

## Screenshots
- Deliver
<img width="1109" alt="Screenshot 2020-05-22 at 11 01 16" src="https://user-images.githubusercontent.com/35291291/82650838-910fc000-9c1b-11ea-8b32-49d46d3985b0.png">

- Invest
<img width="1113" alt="Screenshot 2020-05-22 at 11 01 34" src="https://user-images.githubusercontent.com/35291291/82650859-9b31be80-9c1b-11ea-8256-8c36a233b960.png">
